ui/panels-plugin/src/plugins/time-series-chart/time-series-chart-model.ts

```diff
@@ -30,6 +30,23 @@ export interface TimeSeriesChartOptions {
  */
 export function createInitialTimeSeriesChartOptions(): TimeSeriesChartOptions {
   return {
-    queries: [],
+    queries: [
+      {
+        kind: 'TimeSeriesQuery',
+        spec: {
+          plugin: {
+            kind: 'PrometheusTimeSeriesQuery',
+            spec: {
+              query: 'up',
+            },
+          },
+        },
+      },
+    ],
+    // TODO: configurable legend, change from show_legend to legend.show
```

Comment:
:+1:

I'm probably going to write down some thoughts today, but basically it was on my mind to sync with you about the schema for chart options